src/schedule/schedule-week-view/schedule-week-view.styles.tsx

```diff
@@ -0,0 +1,243 @@
+        return `calc((100% - 14px) * ${$actualWidthPercentage} / 100)`;
+    }};
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    pointer-events: none;
+    overflow: visible;
+    z-index: 1;
+    box-sizing: border-box;
+`;
+
+export const SlotContent = styled(Typography.BodyXS)<{
+    $status: string;
+    $duration: number;
+    $offsetTop: number;
+}>`
+    /* Fill the overlay column width responsively with a small horizontal inset */
+    width: calc(100% - 1px);
+    height: ${({ $duration }) =>
+        $duration
+            ? `${($duration / 30) * CELL_HEIGHT - 1}px`
+            : `${CELL_HEIGHT - 1}px`};
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    padding: 2px 4px;
+    position: absolute;
+    top: ${({ $offsetTop }) => $offsetTop || 0}px;
+    margin-top: 0;
+    font-weight: 500;
+    border-radius: ${Radius["sm"]};
+    box-sizing: border-box;
+
+    background: ${({ $status }) => {
+        switch ($status) {
+            case "pending":
+                return css`
+                    repeating-linear-gradient(
+                        135deg,
+                        ${Colour["bg-warning"]},
+                        ${Colour["bg-warning"]} 5px,
+                        ${Colour["bg-warning-hover"]} 5px,
+                        ${Colour["bg-warning-hover"]} 10px
+                    )
+                `;
+            case "blocked":
+                return Colour["bg-inverse-subtle"];
+            case "available":
+                return Colour["bg-success-hover"];
+            case "booked":
+                return Colour["bg-primary-subtler"];
+            default:
+                return Colour["bg"];
+        }
+    }};
+
+    color: ${({ $status }) =>
+        $status === "blocked" ? Colour["text-inverse"] : "inherit"};
+
+    border-left: ${Border["width-040"]} solid
+        ${({ $status }) =>
+            $status === "available" ? Colour["icon-success"] : "none"};
+`;
+
+export const SlotServiceName = styled.span`
+    display: block;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: 100%;
+`;
+
+export const SlotAvailability = styled.span`
+    display: block;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: 100%;
+`;
+
+export const HiddenColumnsButton = styled.div<{
+    $heightPercentage?: number;
+    $isMultiple?: boolean;
+}>`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    color: ${Colour["text-primary"]};
+    min-height: ${({ $heightPercentage }) =>
+        $heightPercentage ? `${$heightPercentage}%` : `${CELL_HEIGHT}px`};
+    height: ${({ $heightPercentage }) =>
+        $heightPercentage ? `${$heightPercentage}%` : "auto"};
+    pointer-events: auto;
+    &:hover {
+        border-radius: ${Radius["sm"]};
+        box-shadow: 0 2px 8px 0 ${Colour["border"]};
```

Comment:
use the new shadow token: https://designsystem.life.gov.sg/react/index.html?path=/docs/foundations-shadow-default--docs